draft-opsarea-rfc5706bis.md

```diff
@@ -870,6 +870,25 @@ Information Models and Data Models
      the L3VPN Service Model (L3SM) {{?RFC8299}} and the L2VPN Service Model (L2SM)
      {{?RFC8499}}.
 
+  A common challenge in YANG model development lies in defining the
```

Comment:
```suggestion
  A common challenge in YANG data model development lies in defining the
```

Reply:
Do we want to limit it to data model here?  We're starting to expand to info models, too.  Maybe data and information model development?

Reply:
I'm afraid DM makes more sense for this specific text proposal as the text reasons about "common xxx". We don't have sufficient practice for YANG-based IMs.

If we extend to IMs, then we need to revisit the full sentence.